src/model/Shape.java

```diff
@@ -0,0 +1,28 @@
+package model;
+
+
+public abstract class Shape implements ShapesProperties {
+	
+	/************************************
```

Comment:
Shape is an abstract class. Why is it also an interface according to the class diagramm picture?